Update manual instruction for postgres and neo4j (tested)

cedricosei opened the pull request:
Hi Jonathan, 
as discussed in teams I am sending this pull request specific to the first part of the updates of task 2. The second part containing the update for mongodb and GraphDB should follow shortly after this week. 
Kind regards, 
C

cedricosei committed:
Structural change of setup README

Structural change of specific database instructions. To be further extended

cedricosei committed:
Update postgres and neo4j manual instruction code

cedricosei committed:
Update postgres and neo4j manual instruction code

cedricosei committed:
Merge branch 'jf87:main' into main

jf87 commented:
Can you add more details about the scripts for Postgres and Neo4j? What are the parameters that need to be used? Maybe add some example parameterization? In Postgres, does it work out of the box, or do we need to create a custom schema?

cedricosei committed:
update of description of postgres and neo4j according to comment from pull request

cedricosei committed:
Merge remote-tracking branch 'origin/main'

cedricosei commented:
I added some more description for both postgres and neo4j regarding the running of the script. Simplified as much as possible. And yes both scripts work out of the box.
